v2.0/admin-ui-access-and-navigate.md

```diff
@@ -107,6 +84,58 @@ Down Since | How long the node has been down.
 
 When you decommission a node, CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down. See [Remove Nodes](remove-nodes.html) for more information.
 
+### Node Map (Enterprise)
+
+The **Node Map** is an [enterprise-only](enterprise-licensing.html) feature that gives you a visual representation of the geographical configuration of your cluster. To set up and enable the Node Map, see [Enable Node Map](enable-node-map.html). The Node Map consists of the following components:
```

Comment:
I'd cut out the second sentence and move it after the component sections, with some edits:

> For guidance on enabling and using the node map, see ...